capstone-rs/examples/recursive.rs

```diff
@@ -0,0 +1,87 @@
+use std::collections::{HashMap, VecDeque};
+use std::env;
+use std::fs;
+use std::process;
+
+use object::{Object, ObjectSection, SectionKind};
+
+use capstone;
+use capstone::arch::x86::X86Insn;
+use capstone::prelude::*;
+
+fn main() {
+    let args: Vec<_> = env::args().collect();
+    if args.len() != 2 {
+        eprintln!("Usage: {} <file>", args[0]);
+        process::exit(-1);
+    }
+
+    let buf = if let Ok(bd) = fs::read(&args[1]) {
+        bd
+    } else {
+        eprintln!("cannot read file");
```

Comment:
Since this is an example program and not a library, you can just use `.expect("my message")` on `Option`/`Result` instead. This will keep the example shorter and easier to read.